initializr-generator-beans-xml-config/src/test/java/eu/xenit/alfred/initializr/generator/beans/config/SpringXmlConfigWriterTest.java

```diff
@@ -203,7 +203,7 @@ public void testWritePropsProperty() throws IOException {
 ////            <list>
 ////                <props>
 ////                    <prop key="engineId">activiti</prop>
-////                    <prop key="location">alfresco/module/test-maven-sdk-platform/workflow/sample-process.bpmn20.xml</prop>
+////                    <prop key="locationStrategy">alfresco/module/test-maven-sdk-platform/workflow/sample-process.bpmn20.xml</prop>
```

Comment:
automatically refactored by intellij when refactoring this? [link](https://github.com/xenit-eu/start.xenit.eu/pull/20/commits/d2873184a802f2bf193d300970327309d60f817c#diff-b19a3765a6c4b0eae504a9beb4d28b09R20)

Reply:
already fixed in later commit